src/Classes/Particle/Repulser.ts

```diff
@@ -125,21 +124,21 @@ export class Repulser {
         const dyMouse = particle.position.y - mousePos.y;
         const distMouse = Math.sqrt(dxMouse * dxMouse + dyMouse * dyMouse);
         const normVec = {x: dxMouse / distMouse, y: dyMouse / distMouse};
-        const repulseRadius = options.interactivity.modes.repulse.distance;
+        const repulseRadius = container.retina.repulseModeDistance;
         const velocity = 100;
         const repulseFactor = Utils.clamp((1 - Math.pow(distMouse / repulseRadius, 2)) * velocity, 0, 50);
         const pos = {
             x: particle.position.x + normVec.x * repulseFactor,
             y: particle.position.y + normVec.y * repulseFactor,
         };
-        const outMode = options.particles.move.out_mode;
+        const outMode = options.particles.move.outMode;
 
-        if (outMode === OutMode.bounce || outMode === OutMode.bounceVertical) {
-            if (pos.x - particle.radius > 0 && pos.x + particle.radius < container.canvas.dimension.width) {
+        if (outMode === OutMode.bounce || outMode === OutMode.bounceVertical || outMode === OutMode.bounceHorizontal) {
+            if (outMode === OutMode.bounceVertical || (pos.x - particle.radius > 0 && pos.x + particle.radius < container.canvas.dimension.width)) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Exceeds maximum line length of 120](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5136254)